build-config/resources/Ballerina.toml

```
@@ -0,0 +1,19 @@
+[package]
+distribution = "2201.9.2"
+org = "ballerinax"
+name = "aws.marketplace.mpe"
+version = "@toml.version@"
+license= ["Apache-2.0"]
+authors = ["Ballerina"]
+keywords = ["IT Operations/Databases", "Cost/Freemium"]
```

Are these keywords correct? 
```suggestion
keywords = ["IT Operations/Databases", "Cost/Freemium"]
```

---

Fixed with [4a8455523de49bd042c328efd95710ab28b77f8a](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/1/commits/4a8455523de49bd042c328efd95710ab28b77f8a)